ModuleChangelog.md

```diff
@@ -1,6 +1,6 @@
-## 2.0.1
+## 2.0.2
```

Comment:
Just bumping the version number since we didn't publish on PSGallery / release last merge